### ISSUE TITLE
[FeatureRequest] Go home

### ISSUE THREAD

**Issue by andreasMikk** [PokemonGoF/PokemonGo-Bot#3773]
Hi,

Sorry if this is not the right place to post this but just wanted to share an idea.
I would like to suggest a Go home then quit feature.
If you are like me you will bot in your own town so you can stop the bot and wait a reasonable amount of time before you open the app at home.
It would be a nice feature if you can add your home location and get the bot to walk home and then quit.

Thanks :)

Regards
Andreas M

**Comment by mico** [comment]
agree

**Comment by RedViper9** [comment]
For the time being you can get behavior similar to this using the MoveToMapPokemon feature. You will need to set "update_map": false, and you may need to increase the range depending on how far away your bot is from "home". If you start the PokemonGo-Map with your home as it's coordinates, your bot should go to the area of your home and catch pokemon on the way. If you set the search radius "steps" to something small, the bot won't wander too far away from your house.

**Comment by DimaVIII** [comment]
You can use this app on ypur iPhone for GPS-spoofing. No Jailbreak needed. 
https://www.youtube.com/watch?v=VVChSH45EQg

**Comment by k4n30** [comment]
Please move your feature request/s to our feature hub - http://feathub.com/PokemonGoF/PokemonGo-Bot

**Comment by jrjhealey** [comment]
I like this idea a lot, for those people who only want to bot a little and still play legitimately in the intervening time, or to open the app and check up on what the bot has been up to.

**Comment by k4n30** [comment]
https://github.com/PokemonGoF/PokemonGo-Bot/issues/3843#issuecomment-239708680

**Comment by k4n30** [comment]
going to consider this resolved by #4443

**Comment by lntn** [comment]
I can't see how wake_up_at_location can help on this request. What I need is actually "sleep at location" whereby the bot auto move to a certain location then stop

**Comment by RedViper9** [comment]
@Intn Let me know if I'm right in this.
@k4n30 
The request is to have the bot come back to a home area, presumably to avoid softbans resulting from switching from bot to mobile app. The wake_up_at_location option may resolve that if the bot is interrupted after sleep, but provides no benefit (in terms of avoiding softbans) compared with disabling the bot for a period of time.

**Comment by lntn** [comment]
@RedViper9 you're right, it's to avoid the "teleport" effect.
Without this we will need to wait for a while before we can launch the mobile app to enjoy our catches.

**Comment by lntn** [comment]
@jrjhealey yeah that's true. May I know what is the benefit for botting in US instead of UK?

**Comment by jrjhealey** [comment]
Well I'm sure there are locations in the UK that I could use. I've just heard Santa Monica has good density of Pokestops etc.

There are continent specific catches as well so there is a legitimate reason to hop continents.

**Comment by RedViper9** [comment]
@jrjhealy I don't expect it would be a manditory task, but many people have noted that they bot around their home areas. In any case, there have been a number of requests for such a feature.

For your use, the "go home" function could walk you back to a particularly rich area in Santa Monica if your bot has wandered off, for example.

**Comment by avexus** [comment]
@RedViper9 "update_map": false is not in the sample configure file any more.

**Comment by taimaster043** [comment]
go home should b where u start, if ur in usa and at pier 39, when everything is done, u will end ur bot day at pier 39, even if ur in another country. :)
